apps/frontend/src/routes/register/verify/index.tsx

```diff
@@ -17,11 +18,16 @@ export const onGet: RequestHandler = async ({ cookie, redirect }) => {
   }
 };
 
+const COUNT = 59;
```

Comment:
Consider change this name to a more meaningful name

Reply:
Done